ui/css/oae.signup.css

```diff
@@ -28,38 +36,142 @@ main > h3 {
     padding-bottom: 10px;
 }
 
+#signup-institution-container,
+#signup-options-container {
+    margin: 0 20px;
+    padding: 40px;
+}
+
+#signup-institution,
+#signup-options {
+    margin: auto;
+    max-width: 992px;
+}
+
+
+/**************************
+ ** INSTITUTIONAL SEARCH **
+ **************************/
+
+/** Layout **/
+
+#signup-institution-container {
+    border-bottom: solid 1px #DADADA;
```

Comment:
It appears that this border is lighter than the one between social and create account. The colors should be reversed so the darkest one separates selecting an institution from signing up to the current tenant.